packages/functions/docs/modules/index.md

```diff
@@ -40,16 +43,36 @@ Returns the location information for the incoming request.
 
 [`Geo`](../interfaces/index.Geo.md)
 
+The location information of the request, in this way:
+
+```json
+{
+ "city": "New York",
+ "country": "United States",
+ "flag": "🇺🇸"
+ "countryRegion": "NY",
+ "region": "dev1",
+ "latitude": "40.7128",
+ "longitude": "-74.0060"
+}
+```
+
 #### Defined in
 
-[packages/functions/src/headers.ts:128](https://github.com/vercel/vercel/blob/main/packages/functions/src/headers.ts#L128)
+[packages/functions/src/headers.ts:158](https://github.com/vercel/vercel/blob/main/packages/functions/src/headers.ts#L158)
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (soft-blocking):** use canonical link

This link goes to `main`, which doesn't have this line number.